src/shared/containers/Refile/RefileErrorsForm.jsx

```diff
@@ -105,31 +96,58 @@ class RefileErrorsForm extends Component {
     this.setState({ formFields: {...this.state.formFields, [name]: value} });
   }
 
+  /**
+  * @desc Validates the input
+  * @param {string} date - the string of the input value
+  */
+  isDateValid(date) {
+    if (!date) {
+      return false;
+    }
+
+    const dateArray = date.split('/');
+    // Checks if it has a valid date format. The Regex check if the inputs are digits
+    // and if they have right number of digits
+    const dateMatches = date.match(/^(\d{2})\/(\d{2})\/(?:\d{4})$/);
+
+    if (!dateMatches) {
+      return false;
+    }
+
+    // Checks if the month is valid
+    if (parseInt(dateArray[0], 10) < 1 || parseInt(dateArray[0], 10) > 12) {
```

Comment:
Very, very minor point, makes no difference, but I'd do 

```
let month = parseInt(dateArray[0], 10)
```

More readable, don't have to repeat the `parseInt` with its base-10, and only one thing to change if the regex changes. Obviously makes no practical difference here, but I prefer it for general style.

Reply:
I can update it! Good suggestions!